new-module/liquidations.md

```diff
@@ -0,0 +1,111 @@
+---
+description: Liquidations in Angle _New_ Module
+---
+
+# Liquidations
```

Comment:
Let's also insist on the fact that liquidations are designed to be super borrower friendly

Reply:
We should also mention the stuff about dust and dustCollateral and the fact that liquidating can lead to the creation of bad debt and that this is handled and so on